PROTOCOL.md

```diff
@@ -227,11 +227,12 @@ represented by a space), and the rest of the string being their username.
 > Finish logging in (or renaming) by sending: `/trn USERNAME,0,ASSERTION`
 > where `USERNAME` is your desired username and `ASSERTION` is `data.assertion`.
 
-`|updateuser|USERNAME|NAMED|AVATAR`
+`|updateuser|USERNAME|NAMED|AVATAR|BLOCKPMS|BLOCKCHALLENGES`
```

Comment:
Hm, I would probably want to send the rest via JSON, rather than ossifying them?

Reply:
> I think though it would be better to make a JSON field `SETTINGS` instead of tacking on additional user settings as needed?

Agreed.